src/farmerbot/farmerbot.md

```diff
@@ -426,6 +451,149 @@ Dedicated nodes are managed like any other node. Nodes marked as dedicated can o
 
 ***
 
+
+## Can I run the Farmerbot on a Raspberry Pi?
+
+It is possible to run the Farmerbot on a Pi. That being said, it is experimental and Threefold might never support it officially.
+
+To run the Farmerbot on a Raspberry Pi, read [this guide](https://forum.threefold.io/t/how-to-run-farmerbot-on-a-raspberry-pi/3879/).
+
+***
+
+## I want to set up the Farmerbot for my 3node farm. Should all my nodes have the same twin ID?
+
+The farmerbot can only manage the nodes that you define in the configuration. So, for each node in your farm, you will need to fill in these required attributes:
+
+* id
+  * the id of the node
+* twinid
+  * the twin id of the node
+
+***
+
+## How can I use GraphQL to see if a 3node is up or down in relation with the Farmerbot?
+
+This script will show you all the nodes' status on [GraphQL](https://graphql.grid.tf/graphql):
+
+```
+query MyQuery {
+  nodes {
+    nodeID
+    power {
+      state
+    }
+  }
+}
+```
+
+To find a specific node, write the following with the proper nodeID (here we use 655 as an example):
+
+```
+query MyQuery {
+  nodes(where: {nodeID_eq: 655}) {
+    power {
+      state
+    }
+    nodeID
+  }
+}
+```
+
+***
+
+## When setting up the Farmerbot, I get the error: dependency failed to start: container qa-redis-1 is unhealthy. What can I do?
+
+There's some redis database state stored in a persistent volume. The issue is that the database got corrupted somehow and thus redis is crashing.
+
+To remove the volume, and thus actually get a clean start, you can do this:
+```
+docker compose down -v
+```
+
+***
+
+## Can you run multiple Farmerbots on a single computer?
+
+It's been determined that multiple Farmerbots can be hosted on a single machine. In that case, each Farmerbot will need its own `.env` and `config` files. 
+
+Furthermore, in its current state, each Farmerbot can only control a single twin account, and each twin should only oversee a single farm. 
+
+At a later stage, it should be possible to run multiple farms under a single twin.
+
+***
+
+## With the Farmerbot, how can I set a node to never shut down?
+
+In the config file, set the following: `never_shutdown:true` for the node you want to never shut down.
+
+***
+
+## How long does it take for the Farmerbot to do the periodic wake?
+
+The minimum period between two nodes to be waken is currently 5 minutes.
+So every 5 minutes a new node wakes up.
+
+Once all nodes are awaken, they all shut down at the same time, except the node that stays awaken to wake up the other during the next periodic wake.
+
+A future release will permit farmers to specify how much nodes will be powered on at the same time during a periodic wakeup. This will enable faster periodic wake but might cause spikes in electricity consumption. Farmers will thus need to adjust with their own setup.
```

Comment:
OK. I removed
"A future release will permit farmers to specify how much nodes will be powered on at the same time during a periodic wakeup. This will enable faster periodic wake but might cause spikes in electricity consumption. Farmers will thus need to adjust with their own setup."